src/main/java/frc/robot/commands/testing/TestCommand.java

```diff
@@ -202,6 +202,8 @@ public void end(boolean interrupted) {
     shooterSubsystem.stop();
     turretSubsystem.stop();
     intakeSubsystem.stop();
-    turretSubsystem.prepareToIntake();
+    turretSubsystem.stopPitch();
+    turretSubsystem.stopYaw();
+    turretSubsystem.runRollers(RotationsPerSecond.zero());
```

Comment:
Line 203 calls `turretSubsystem.stop()`, so I don't think you need lines 205-207, the turret will already be stopped.

(Side note, `runRollers(RotationsPerSecond.zero())` doesn't turn the rollers off, it actively drives and holds the velocity at zero. `stopRollers()` is the method to turn the roller motors off.)